index.js

```diff
@@ -660,20 +659,17 @@
   //. false
   //. ```
 
-  //# Either#extend :: Either a b ~> (b -> b) -> Either a b
+  //# Either#extend :: Either a b ~> (Either a b -> b) -> Either a b
   //.
   //. Takes a function and returns `this` if `this` is a Left; otherwise it
   //. returns a Right whose value is the result of applying the function to
-  //. this Right's value. `Either#extend` is a restricted form of
-  //. [`Either#map`](#Either.prototype.map): the function provided must
-  //. return a value of the same type as its input (`Either#map` does not
-  //. have this restriction).
+  //. `this`.
   //.
   //. ```javascript
   //. > S.Left('Cannot divide by zero').extend(R.inc)
```

Comment:
I believe the 79-character maximum line length is no longer enforced by `make lint`. It seems reasonable to exceed the limit in this case.